src/panels/config/integrations/integration-panels/ozw/dialog-ozw-refresh-node.ts

```diff
@@ -0,0 +1,274 @@
+import {
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  internalProperty,
+  TemplateResult,
+  PropertyValues,
+  css,
+} from "lit-element";
+import "../../../../../components/ha-code-editor";
+import "../../../../../components/ha-circular-progress";
+import { createCloseHeading } from "../../../../../components/ha-dialog";
+import { haStyleDialog } from "../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../types";
+import { OZWRefreshNodeDialogParams } from "./show-dialog-ozw-refresh-node";
+
+import {
+  fetchOZWNodeMetadata,
+  OZWDeviceMetaData,
+  OZWDevice,
+  nodeQueryStages,
+} from "../../../../../data/ozw";
+
+@customElement("dialog-ozw-refresh-node")
+class DialogOZWRefreshNode extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property()
+  private node_id: number | undefined;
```

Comment:
private should be internal and prefixes with an underscore
```suggestion
  @internalProperty()
  private _node_id: number | undefined;
```

Reply:
This should also be on the same line no?
```suggestion
  @interalProperty() private _node_id: number | undefined;
```